### ISSUE TITLE
2.7.3 version?

### ISSUE THREAD

**Issue by dark-passages** [keepassxreboot/keepassxc#8632]
Ok, today I got a message from KeePassXC 2.7.1 that the 2.7.3 version is available. (no clue why the 2.7.2 update never showed up) So I clicked on the convenient link in the message to download and install the new version.
After installing the 2.7.3 MSI file the about screen still shows 2.7.1? Check for updates still shows there is a newer version?
So I removed the 2.7.1 version completely and downloaded & installed the 2.7.3 MSI file again. Still 2.7.1 in the about screen! So remove everything again and try to install from the Microsoft store, the download failed, retry and restart several times and it keeps failing. Also downloaded the portable version, but seems to have the same problem.
What should I do to get this to work again?

**Comment by treee111** [comment]
If you want the 2.7.3 version download it from the [release page](https://github.com/keepassxreboot/keepassxc/releases/tag/2.7.3)

Other download-locations/pages might not been updated with version 2.7.3 because this is brand new from yesterday. 2.7.2 was kind of skipped because there was a crucial thingy which was fixed in 2.7.3. Don't worry about 2.7.2.

**Comment by droidmonkey** [comment]
Microsoft store hasn't updated yet, Microsoft takes a long time to approved an update. Try downloading the MSI again, you may have gotten ahead of us on getting the webpage up to speed. Make sure your download says 2.7.3 in the file name.

**Comment by dark-passages** [comment]
> Make sure your download says 2.7.3 in the file name.

I got the downloads from the official download page. On the page it stated 2.7.3 and the filename also shows 2.7.3.

So I removed the app, removed the folder manually (not done by uninstall process?) and remove a bunch of registry keys left behind. Then did a new download and install... finally 2.7.3 is installed. But this subject is already closed?
